### PR TITLE
Fix in multi tenancy doc

### PR DESCRIPTION
**Pull request by mathieuprog**

### CODE REVIEW

File: guides/howtos/Multi tenancy with foreign keys.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ With these changes, we will always set the `org_id` fields in our queries, unles
 
 Let's expand our data domain little a bit.
 
-So far we have assumed there is an organization schema. However, instead of naming its primary key `id`, we will name it `org_id`, so `Repo.get(Org, org_id: 13)` just works:
+So far we have assumed there is an organization schema. However, instead of naming its primary key `id`, we will name it `org_id`, so `Repo.one(Org, org_id: 13)` just works:
```
Comment (v0idpwn) [inline guides/howtos/Multi tenancy with foreign keys.md:94]: I believe it should be `Repo.get_by/2`, no?

`Repo.one/2` second argument receives options, not a select field

Comment (josevalim) [reply guides/howtos/Multi tenancy with foreign keys.md:94]: Yup, get_by. :)

Comment (mathieuprog) [reply guides/howtos/Multi tenancy with foreign keys.md:94]: No, it's `Repo.one/2`. My PR is correct 😁 `org_id: 13` is a repo option.

Comment (v0idpwn) [reply guides/howtos/Multi tenancy with foreign keys.md:94]: Oh, I see, you're correct

**Review by v0idpwn — APPROVED** [review]

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**